doc/rst/tools/mason/mason.rst

```diff
@@ -93,8 +93,38 @@ For example, after ``mason run [ options ]``, the project directory appears as s
 	  MyPackage
 	
    
+Projects can also be multiple files. Mason handles this by forwarding all arguments
```

Comment:
Note that projects can be multiple files without getting compiler flags involved. `-M` is actually not necessary for the example below, but would be example for a case more like:

```
MyPackage/
  Mason.toml
  Mason.lock
  src/
  MyPackage.chpl
  utils/
    MyPackageUtils.chpl
```

Reply:
I think it'd be good to document the handling of compiler flags / execution flags as a standalone topic. The usage of `-M` to use modules in subdirectories seems more FAQ-ish than appropriate for documentation. An FAQ at the bottom of the docs wouldn't be a bad idea.

Reply:
I think this is too important of a topic to push to the bottom of the documentation before we have compile flags in ``Mason.toml``.